src/fieldImpl.cc

```diff
@@ -106,44 +94,48 @@ Field::addExt(lsst::afw::image::Wcs const& wcs_,
     /*
      * We're going to fake psfex's wcsstruct object.  We only need enough of it for field_locate
      */
-    PsfUnpack wcsUnpacked(wcs_);
-    struct wcsprm const* wcsPrm = wcsUnpacked.getWcsInfo();
     QMALLOC(impl->wcs[impl->next], wcsstruct, 1);
     wcsstruct *wcs = impl->wcs[impl->next];
     
-    wcs->naxis = wcsPrm->naxis;
+    wcs->naxis = 2;
     wcs->naxisn[0] = naxis1;
     wcs->naxisn[1] = naxis2;
-    
+
+    auto const crval = wcs_.getSkyOrigin();
+    // crpix using the FITS standard = pixel origin using the LSST standard + 1
+    auto const crpix = wcs_.getPixelOrigin() + afw::geom::Extent2D(1, 1);
+    auto const cdMatrix = wcs_.getCdMatrix();
+    std::string const cunit("DEG");
+    auto metadata = wcs_.getFitsMetadata();
     for (int i = 0; i != wcs->naxis; ++i) {
-        strncpy(wcs->ctype[i], wcsPrm->ctype[i], sizeof(wcs->ctype[i]) - 1);
-        strncpy(wcs->cunit[i], wcsPrm->cunit[i], sizeof(wcs->cunit[i]) - 1);
-        wcs->crval[i] = wcsPrm->crval[i];
-        
-        wcs->cdelt[i] = wcsPrm->cdelt[i];
-        wcs->crpix[i] = wcsPrm->crpix[i];
-        wcs->crder[i] = wcsPrm->crder[i];
-        wcs->csyer[i] = wcsPrm->csyer[i];
-        wcs->crval[i] = wcsPrm->crval[i];
+        auto ifits = i + 1;
```

Comment:
I'm not sure what all these wcs parameters mean here and below.  I guess they are safe to be set to zero?

Reply:
I think so. I could not find any psfex documentation for this structure, so I set what fields I could based on the [wcslib wcsprm docs](http://www.atnf.csiro.au/people/mcalabre/WCS/wcslib/structwcsprm.html#a15485177ea8bbacefc29a5a5cba98c8f). crdr and csyer are uncertainties, which we have no way of knowing, so I thought 0 was reasonable.